cpp/tests/CMakeLists.txt

```diff
@@ -400,167 +379,182 @@ ConfigureTest(TRIANGLE_COUNT_TEST community/triangle_count_test.cpp)
 # - MG tests --------------------------------------------------------------------------------------
 
 if(BUILD_CUGRAPH_MG_TESTS)
-
     ###############################################################################################
     # - find MPI - only enabled if MG tests are to be built
     find_package(MPI REQUIRED)
 
+    add_library(cugraphmgtestutil STATIC
+                utilities/device_comm_wrapper.cu
+                utilities/mg_utilities.cpp)
+
+    set_property(TARGET cugraphmgtestutil PROPERTY POSITION_INDEPENDENT_CODE ON)
+
+    target_include_directories(cugraphmgtestutil
+        PRIVATE
+        "${CMAKE_CURRENT_SOURCE_DIR}/../../thirdparty/mmio"
+        "${CMAKE_CURRENT_SOURCE_DIR}/../include"
+        "${CMAKE_CURRENT_SOURCE_DIR}"
+    )
+
+    target_link_libraries(cugraphmgtestutil
+        PUBLIC
+            cugraph::cugraph
+            NCCL::NCCL
+            MPI::MPI_CXX
+    )
+
     # Set the GPU count to 1.  If the caller wants to execute MG tests using
     # more than 1, override from the command line using -DGPU_COUNT=<gpucount>
     if (NOT DEFINED GPU_COUNT)
       set(GPU_COUNT "1")
     endif()
 
-    if(MPI_CXX_FOUND)
```

Comment:
I'm curious why this if/else block with the informative error message is being removed.

Reply:
Since we requie MPI to be found, that will error out before this if branch is reached.
To be extra sure, I have listed `CXX` as a required COMPONENT to `find_package(MPI)`